INSTALL.md

```diff
@@ -159,8 +150,9 @@ Python Interface
 The Python interface is provided via SWIG (Simple Wrapper and
 Interface Generator) and an additional level of manual wrappers (in faiss/python/faiss.py).
 
-SWIG generates two wrapper files: a Python file (`faiss/python/swigfaiss.py`) and a
-C++ file that must be compiled to a dynamic library (`faiss/python/_swigfaiss.so`).
+SWIG generates two wrapper files: a Python file `swigfaiss.py` and a
+C++ file that must be compiled to a dynamic library (`_swigfaiss.so`).
+There is an AVX2 variant of the files, suffixed with `_avx2`. 
```

Comment:
This part is not really relevant to installing, and is probably more confusing than helpful to new users.

Reply:
Will move it to the packaging section.